### DISCUSSION TITLE
Moes BHT-002-GCLZB - set schedule from Home Assistant

### DISCUSSION THREAD

**Discussion by adorobis** [Koenkk/zigbee2mqtt#14973] [Devices]
Hello, is there a simple way to set the thermostat schedule from Home Assistant? Or would I need to publish the MQTT message as per definition for this device with some custom script/automation in HA?

**Comment by Bandit13** [comment]
In documentation https://www.zigbee2mqtt.io/devices/BHT-002-GCLZB.html you can use set topic zigbee2mqtt/FRIENDLY_NAME/set

**Reply by adorobis** [reply]
Yes, I have seen that already and I've played with it with MQTT Explorer before posting the question. Works very well. I was just wondering if there is any feature in HA (build in or a custom integration etc.) that would make this simpler.

**Comment by adorobis** [comment]
I guess investing in it does not make much sense as this can be now nicely edited in the zigbee2mqtt UI:
![image](https://user-images.githubusercontent.com/19752098/205945983-b010fed5-796a-4ed1-9854-2081ade5b98b.png)
